File: python/python3/CSharp/Python3ParserBase.cs
using System;
using System.Collections.Generic;
using System.IO;
using Antlr4.Runtime;

public abstract class Python3ParserBase : Parser
{
	protected Python3ParserBase(ITokenStream input)
			: base(input)
	{
	}

	protected Python3ParserBase(ITokenStream input, TextWriter output, TextWriter errorOutput)
			: base(input, output, errorOutput)
	{
	}

	public bool CannotBePlusMinus()
	{
		return true;
	}

	public bool CannotBeDotLpEq()
	{
		return true;
	}
}

File: python/python3/JavaScript/Python3ParserBase.js
import antlr4 from 'antlr4';
import Python3Parser from './Python3Parser.js';

export default class Python3ParserBase extends antlr4.Parser {

    constructor(input) {
        super(input);
    }

    CannotBePlusMinus() {
        return true;
    }   

    CannotBeDotLpEq() {
        return true;
    }   
}
